hasona23.ShiftsLogger/ShiftsLogger/ShiftsLoggerContext.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ShiftsLoggerAPI.Models;
+
+
+namespace ShiftsLoggerAPI
+{
+
+    public class ShiftsLoggerContext : DbContext
+    {
+        public DbSet<Worker> Workers { get; set; }
+        public DbSet<Shift> Shifts { get; set; }
+
+        public const string ConnectionString = "Server=(localdb)\\mssqllocaldb;Database=ShiftsLoggerDB;Trusted_Connection=True";
```

Comment:
Still using hard-coded connection strings?
I thought we learned how to use configuration files in the `coding-tracker` project?